tests/integration/src/setup.rs

```diff
@@ -68,7 +68,7 @@ pub fn setup_new_env_with_config(config: SetupConfig) -> TestEnv {
         .with_nns_subnet()
         .with_application_subnet()
         .build();
-    env.set_time(SystemTime::now());
+    env.set_time(SystemTime::now() - Duration::from_secs(24 * 60 * 60));
```

Comment:
would be great to add a comment explaining the reason to put the time in the past

Reply:
Added a comment.